topics/introduction/tutorials/galaxy-intro-strands/tutorial.md

```diff
@@ -112,8 +112,7 @@ The Galaxy interface consists of three main parts. The available tools are liste
 > * If this is the first time you have used this Galaxy server, then your history panel will look like the image above.
 >
 > * If your history has datasets in it (there will be one or more green boxes, *datasets*, there) then create a new history:
->     1. Click the **gear icon** at the top of the history panel
->     2. Select the option **Create New** from the menu
+>     * Click the **plus icon** to create a new history
```

Comment:
Can you use the `snippets/create_new_history.md` snippet?

Reply:
I've inserted this together with short text above to tell people what they should be doing - the snippet can then explain how to do so.